File: src/main/java/no/hvl/tk/visual/debugger/settings/DebuggingVisualizerOption.java
package no.hvl.tk.visual.debugger.settings;

public enum DebuggingVisualizerOption {
  /** Web ui visualizer using javascript and the browser. */
  WEB_UI,
  /** Embedded visualizer using plant uml. */
  EMBEDDED;

  @Override
  public String toString() {
    return switch (this) {
      case WEB_UI -> "Browser visualizer";
      case EMBEDDED -> "Embedded visualizer (no interaction)";
    };
  }
}

File: src/main/java/no/hvl/tk/visual/debugger/settings/DebuggingVisualizerOption.kt
package no.hvl.tk.visual.debugger.settings

enum class DebuggingVisualizerOption {
  /** Web ui visualizer using javascript and the browser. */
  WEB_UI,

  /** Embedded visualizer using plant uml. */
  EMBEDDED;

  override fun toString(): String {
    return when (this) {
      WEB_UI -> "Browser visualizer"
      EMBEDDED -> "Embedded visualizer (no interaction)"
    }
  }
}
